### PR TITLE
Explain the importance of checkpointing for backups (#1793)

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1811.surge.sh

Updated pages:
- [Back up an online database](https://neo4j-docs-operations-1811.surge.sh/operations-manual/4.4/backup-restore/online-backup)

File: modules/ROOT/pages/backup-restore/online-backup.adoc [hunk 1]
```diff
@@ -217,6 +217,22 @@ The error codes include details of what error was encountered.
...
+The backup cannot proceed until the checkpoint finishes.
+
+While the server is checkpointing, the backup job receives no data, which may lead to the backup timeout.
+To extend the backup timeout, modify the xref:configuration/configuration-settings.adoc#config_dbms.cluster.catchup.client_inactivity_timeout[`dbms.cluster.catchup.client_inactivity_timeout`] setting, which restricts the network inactivity.
+It controls the timeout duration of the catchup protocol, which is the underlying protocol of multiple catchup processes, including backups.
+
+You can also tune up xref:configuration/configuration-settings.adoc#_checkpoint_settings[the Checkpoint settings] or check that your disks are performant enough to handle the load. 
+For more information, see xref:performance/disks-ram-and-other-tips.adoc#performance-checkpoint-iops-limit[Checkpoint IOPS limit].
+
+To read more about checkpointing, see xref:database-internals/checkpointing.adoc[Database internals -> Checkpointing and log pruning].
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/online-backup.adoc:233]:
```suggestion
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1811.surge.sh

Updated pages:
- [Back up an online database](https://neo4j-docs-operations-1811.surge.sh/operations-manual/4.4/backup-restore/online-backup)

File: modules/ROOT/pages/backup-restore/online-backup.adoc [hunk 2]
```diff
@@ -217,6 +217,21 @@ The error codes include details of what error was encountered.
...
 == Online backup configurations
 
+[[online-backup-checkpoints]]
+=== Checkpointing
+
+When a full backup is requested, it always triggers a checkpoint.
+The backup cannot proceed until the checkpoint finishes.
+
+While the server is checkpointing, the backup job receives no data, which may lead to the backup timeout.
+To extend the backup timeout, modify the xref:configuration/configuration-settings.adoc#config_dbms.cluster.catchup.client_inactivity_timeout[`dbms.cluster.catchup.client_inactivity_timeout`] setting, which restricts the network inactivity.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/online-backup.adoc:227]:
```suggestion
To extend the backup timeout, modify the xref:configuration/configuration-settings.adoc##config_causal_clustering.catch_up_client_inactivity_timeout[`causal_clustering.catch_up_client_inactivity_timeout`] setting, which restricts the network inactivity.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1811.surge.sh

Updated pages:
- [Back up an online database](https://neo4j-docs-operations-1811.surge.sh/operations-manual/4.4/backup-restore/online-backup)

**Pull request merged**